app/communities-configs/polygon/index.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { css } from 'styled-components';
+
+import CustomSubHeader from 'components/CustomSubHeaders/CustomSubHeader';
+import CustomMobileSubHeader from 'components/CustomSubHeaders/CustomMobileSubHeader';
+
+import favicon from '!file-loader?name=[name].[ext]!images/favicon-telos.ico';
+import PolygonLogo from './images/polygon-logo.svg?inline';
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+const domainName = 'https://polygon.technology/';
+const domainDocs = 'https://docs.polygon.technology/';
+const domainBlog = 'https://blog.polygon.technology/';
```

Comment:
this 3 constants should be extracted from this file to some url config file or something else